extensions/blocks/publicize/editor.scss

```diff
@@ -66,6 +69,12 @@
 		margin-bottom: 13px;
 	}
 
+	.components-button {
+		height: auto;
+		line-height: initial;
+		padding: 6px 8px;
```

Comment:
Per your PR description, we're explicitly wanting to overwrite the top/bottom of an already set `padding:` value. In that case, let's use `padding-top:` and `padding-bottom:` instead of the shorthand. This way it's more obvious to the next person what the intention behind this change was.

Reply:
Crossed my mind, but wasn't sure if that is preferred. Changed it!